newton/utils/render.py

```diff
@@ -25,6 +27,44 @@
 import newton
 
 
+def xform_to_tqs(prim: Usd.Prim, time=Usd.TimeCode.Default()):
```

Comment:
I think you could make this a `@staticmethod` of the USD renderer since it is only used in this class. That way you can also use the `pxr` import from there.

Reply:
Good catch!